packages/runtime/src/components/hooks/useMediaQuery.ts

```diff
@@ -1,33 +1,35 @@
-/* eslint-env browser */
-
-import { useState } from 'react'
-
+import { useSyncExternalStore } from 'react'
```

Comment:
If you have comments here @migueloller that aren't trivial can we chat through this sync?

Reply:
The import should come from `use-sync-external-store/shim` since we need to continue to support React v17